### PR TITLE
GITC-248: display usd spent for orgs/tribes

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This converts ETH spent to USD spent on org/tribe page

### CODE REVIEW

File: app/dashboard/models.py [hunk 1]
```diff
@@ -4127,7 +4127,7 @@ def get_funder_leaderboard_index(self):
     def get_org_leaderboard_index(self):
         return self.get_leaderboard_index('weekly_orgs')
 
-    def get_eth_sum(self, sum_type='collected', network='mainnet', bounties=None):
+    def get_eth_usd_sum(self, sum_type='collected', network='mainnet', bounties=None, in_usd=False):
```
[the remark this answers is not in the document]
Comment (gdixon) [reply app/dashboard/models.py:4130]: +1 - why are we not defaulting to `in_usd=True`?

File: app/dashboard/models.py [hunk 2]
```diff
@@ -4305,7 +4305,7 @@ def to_representation(instance):
             'url': instance.get_relative_url(),
             'position': instance.get_contributor_leaderboard_index(),
             'organizations': instance.get_who_works_with(network=None),
-            'total_earned': instance.get_eth_sum(network=None)
+            'total_earned': instance.get_eth_usd_sum(network=None)
```
Comment (gdixon) [inline app/dashboard/models.py:4308]: Should this be in ETH or USD?

**Review by gdixon — COMMENTED** [review]
Couple of minor questions, but this looks great @chibie!

**Comment by gdixon** [comment]
Any updates on this PR @chibie?

**Comment by ksolo** [comment]
@chibie can you add unit tests for the changes in dashboard models?

**Review by nutrina — APPROVED** [review]
Changes look good to me. 
But I also agree with @ksolo , we should probably unit test this.

**Comment by Jer-Sch** [comment]
This looks good to me pending the unit tests @ksolo and @nutrina mentioned.

File: app/dashboard/models.py [hunk 3]
```diff
@@ -4157,21 +4157,22 @@ def get_funder_leaderboard_index(self):
     def get_org_leaderboard_index(self):
         return self.get_leaderboard_index('weekly_orgs')
 
-    def get_eth_sum(self, sum_type='collected', network='mainnet', bounties=None):
-        """Get the sum of collected or funded ETH based on the provided type.
+    def get_sum(self, sum_type='collected', network='mainnet', bounties=None, currency='usd'):
```
[the remark this answers is not in the document]
Comment (chibie) [reply app/dashboard/models.py:4154]: done

**Pull request merged**